docker/build.sh

```
@@ -42,14 +42,10 @@ readonly SOURCE_URL="https://github.com/viascom/github-maintenance-action.git"
 readonly VENDOR="Viascom Ltd liab. Co"
 readonly LICENSES="MIT"
 readonly REF_NAME=$VERSION
-readonly BASE_IMAGE="viascom/alpine:3.19.0"
+readonly BASE_IMAGE="viascom/ubuntu:24.04.2"
 readonly BASE_DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' "$BASE_IMAGE")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)